packages/plugin-react-navigation/react-navigation.js

```diff
@@ -0,0 +1,57 @@
+const React = require('react')
+
+class Plugin {
```

Comment:
Should this have a real name? I'm not sure if babel/uglify mangles it anyway, but class names get exposed in some places so it could be useful